compiler/control/OMROptions.hpp

```diff
@@ -193,6 +193,7 @@ enum TR_CompilationOptions
    TR_DisableCompilationThread            = 0x00008000 + 3,
    TR_EnableCompilationThread             = 0x00010000 + 3,
    TR_EnableJITaaSHeuristics              = 0x00020000 + 3,
+   TR_EnableJITServerHeuristics           = 0x00020000 + 3,
```

Comment:
Do these new options use the same option bits as existing options intentionally? The only reason I can think of for doing this is to make the new options aliases of the existing ones. If this is the case, it should be documented accordingly in the commit message.

Reply:
Any name with `JITaaS` will be removed. We're doing staged changes. 
First, add the new names and use them in OpenJ9. 
Secondly, remove the names with `JITaaS` in OMR.